DuckDuckGo/Bookmarks/Model/BookmarkDragDropManager.swift

```diff
@@ -49,6 +49,11 @@ final class BookmarkDragDropManager {
         // If the dragged values contain both folders and bookmarks, only validate the move if all objects can be moved.
         case (true, true), (true, nil), (nil, true):
             return .move
+        case (nil, nil):
+            if info.draggingPasteboard.availableType(from: [.URL, .string]) != nil {
```

Comment:
This will allow to drag any string value that‘ll be converted to a bookmark; Can we instead check for the `.URL` available type (and return `.copy` for it), and validate the `.string` can be converted to a `URL` using the `URL.init?(trimmedAddressBarString:)` helper?
Also we need to use this helper at line `85` to correctly convert the `String` to a `URL`: `let draggedURL = URL(trimmedAddressBarString: draggedString) ` instead of `let draggedURL = URL(string: draggedString)`